leader-election-impl/src/main/java/com/palantir/paxos/PaxosProposerImpl.java

```diff
@@ -203,7 +203,7 @@ public int getQuorumSize() {
     }
 
     @Override
-    public String getUUID() {
+    public String getUuid() {
```

Comment:
ah, cool.  Not sure, just wanted us to be conscious of it and I didn't see any discussion about how it was breaking.